utils/string_parsers.py

```diff
@@ -65,15 +65,14 @@ def convert_uptime_string_seconds(uptime):
     Convert uptime strings to seconds. The string can be formatted various ways, eg.
     1 hour, 56 minutes
     '''
-    regex_1 = re.compile(r"((?P<days>\d+) day(s)?,\s+)?((?P<hours>\d+) hour(s)?,\s+)?((?P<minutes>\d+) minute(s)?)")
+    regex_1 = re.compile(r"((?P<weeks>\d+) week(s)?,\s+)?((?P<days>\d+) day(s)?,\s+)?((?P<hours>\d+) hour(s)?,\s+)?((?P<minutes>\d+) minute(s)?)")
```

Comment:
I know but I don't have any example string to know exactly how it should work : (